### ISSUE TITLE
Missing functionality 

### ISSUE THREAD

**Issue by nerzadler** [aziz/tmTheme-Editor#147]
Hi,
I didn't find how to set a color back to no color (full transparent). Is it possible to do or is the functionality missing ?

Thx !
Nerzadler

**Comment by aziz** [comment]
Hovering over the colorbox should show the reset button on the right.
![](http://cl.ly/image/1x1S2s2A2f2e/Screen%20Shot%202015-05-24%20at%203.39.30%20PM.png)

**Comment by nerzadler** [comment]
Hi again, thanks for your answer.
However, it doesn’t work for me (I’m on OsX Firefox). When I hover over the background color box, nothing happens.

**Comment by aziz** [comment]
Could you give me a screenshot? I tested in Firefox (OSX) and it works. Maybe I'm missing something!

**Comment by nerzadler** [comment]
I gave you a screen-shot in my previous email. Here it is again :


My screen shots doesn’t show my mouse however it is actually on the yellow color box.

I don’t understand why it works for you and not me ! I am using the latest version of osX Firefox.

I tried on osX Safari and it was worse : I cannot even select a color because the color window doesn’t show when I click on a color box…

Hope I help,
Loic
